Lua/Indent.tmPreferences

```diff
@@ -10,7 +10,7 @@
 		<key>decreaseIndentPattern</key>
 		<string>^\s*(elseif|else|end|\})\s*$</string>
 		<key>increaseIndentPattern</key>
-		<string>^\s*(else|elseif|for|(local\s+)?function|if|repeat|until|while)\b((?!end).)*$|\{\s*$</string>
+		<string>^\s*(else|elseif|for|(local\s+)?function|if|repeat|until|while)\b((?!\send).)*$|\{\s*$</string>
```

Comment:
I suggest `(?!.*\bend\b)` instead of `((?!\send).)*$`. It is more performant and checks for non-word symbols on both ends of the keyword.

Note that it will still errorneously match inside strings, but this should get rid of all cases not involving strings.